nodes/AssignName.py

```diff
@@ -0,0 +1,2 @@
+for x in range(3):
```

Comment:
This is unexpected. Is `AssignName` not just a child of `Assign`? (In which case, perhaps use the same example as `Assign` for clarity.)